packages/config/config/devices/0x010f/fgms001.json

```diff
@@ -720,7 +762,7 @@
 			"#": "82",
 			"$if": "firmwareVersion >= 3.2",
 			"label": "LED - illuminance for low indicator brightness",
-			"description": "Light intensity level below which brightness of visual indicator is set to 1%.",
+			"description": "When the light intensity (lux) is below this, the brightness of the light is 1%.",
```

Comment:
The unit should be specified using the `"unit"` property, not as part of the description.